src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioSolutionSizeTracker.cs

```diff
@@ -34,18 +37,10 @@ public SolutionSizeTracker()
         /// lazy and very cheap on answering that question.
         /// </summary>
         public long GetSolutionSize(Workspace workspace, SolutionId solutionId)
-        {
-            var service = workspace.Services.GetService<IPersistentStorageLocationService>();
-            return service.IsSupported(workspace)
-                ? _tracker.GetSolutionSize(solutionId)
-                : -1;
```

Comment:
this used to have different behavior depending on if this was the VSWorkspace or not.  So i just actually supported this through our normal pattern of using an IWorkspaceService and having a special version for the normal case and the VS case.

Note: to better understand how this works:

1. there is one system that checks 'solution size'.  Specifically the persistence storage system (i.e. the sqlite cache).
2. for the remote case, this is *not* used.  in OOP we *always* persist to sqlite, no matter what the solution size.
3. in the in-proc VS case we use this to decide if it's worth it to enable the persistence cache or not.  (i'd prefer we not do this, but that's the current system, so i'm preserving it).  As such, i'm only having the SolutionSizeTracker actually run in the one domain it is actually checked (i.e. VS).